app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,12 @@
+<resources>
+
+    <string name="app_name">YouTube AdAway</string>
+    <string name="module_active">¡está activo!</string>
```

Comment:
I am not familiar with the app, but I think this is not a sentence in itself but rather used to form more complex sentences with some of the other strings (ex: "YouTube AdAway is active!", having the "i" in "is" lowercase also suggests the same). If so, I think the opening exclamation mark should not be here:

```suggestion
    <string name="module_active">está activo!</string>
```